README.md

```diff
@@ -14,74 +14,25 @@ the packages in your project and update them when a stable tag is out.
 
 # Installation
 
-    go get -d github.com/Azure/azure-sdk-for-go/management
-
-# Usage
-
-Read Godoc of the repository at: http://godoc.org/github.com/Azure/azure-sdk-for-go/
-
-The client currently supports authentication to the Service Management
-API with certificates or Azure `.publishSettings` file. You can 
-download the `.publishSettings` file for your subscriptions
-[here](https://manage.windowsazure.com/publishsettings).
-
-### Example: Creating a Linux Virtual Machine
-
-```go
-package main
-
-import (
-	"encoding/base64"
-	"fmt"
-
-	"github.com/Azure/azure-sdk-for-go/management"
-	"github.com/Azure/azure-sdk-for-go/management/hostedservice"
-	"github.com/Azure/azure-sdk-for-go/management/virtualmachine"
-	"github.com/Azure/azure-sdk-for-go/management/vmutils"
-)
-
-func main() {
-	dnsName := "test-vm-from-go"
-	storageAccount := "mystorageaccount"
-	location := "West US"
-	vmSize := "Small"
-	vmImage := "b39f27a8b8c64d52b05eac6a62ebad85__Ubuntu-14_04-LTS-amd64-server-20140724-en-us-30GB"
-	userName := "testuser"
-	userPassword := "Test123"
-
-	client, err := management.ClientFromPublishSettingsFile("path/to/downloaded.publishsettings", "")
-	if err != nil {
-		panic(err)
-	}
-
-	// create hosted service
-	if err := hostedservice.NewClient(client).CreateHostedService(hostedservice.CreateHostedServiceParameters{
-		ServiceName: dnsName,
-		Location:    location,
-		Label:       base64.StdEncoding.EncodeToString([]byte(dnsName))}); err != nil {
-		panic(err)
-	}
-
-	// create virtual machine
-	role := vmutils.NewVMConfiguration(dnsName, vmSize)
-	vmutils.ConfigureDeploymentFromPlatformImage(
-		&role,
-		vmImage,
-		fmt.Sprintf("http://%s.blob.core.windows.net/sdktest/%s.vhd", storageAccount, dnsName),
-		"")
-	vmutils.ConfigureForLinux(&role, dnsName, userName, userPassword)
-	vmutils.ConfigureWithPublicSSH(&role)
-
-	operationID, err := virtualmachine.NewClient(client).
-		CreateDeployment(role, dnsName, virtualmachine.CreateDeploymentOptions{})
-	if err != nil {
-		panic(err)
-	}
-	if err := client.WaitForOperation(operationID, nil); err != nil {
-		panic(err)
-	}
-}
-```
+- [Install Go 1.7](https://golang.org/dl/).
+
+- Go get the SDK:
+
+`go get -d github.com/Azure/azure-sdk-for-go`
+
+- For vendoring dependencies, Azure SDK for Go uses [glide](https://github.com/Masterminds/glide). If you haven't already, install glide. Navigate to the SDK directory and install the dependencies.
+
+`cd azure-sdk-for-go`
```

Comment:
use a triple backtick code block for these two commands and if you add a $ prompt, it may look better